learn/teams/overview.md

```diff
@@ -1,22 +1,41 @@
 ---
-title: Teams Overview
+title: "Overview"
 id: ""
 ---
 ---
 
-Teams are groups of organization members that reflect your organization and organize your group’s work. On a team's page, you can 
+**Teams** are groups of organization members that reflect your organization and organize your group’s work. Teams are designed and intended to be administered by one or more members. On a team's page, you can 
```

Comment:
Can we SILO vs Team. Its basically not a different product. its differnt subscription model.
Link to Subscription in the Website ???
What he can't get ? Customer may expect he can access service behind vpn..  
Vs WME ? No Infra setup etc ?????